scripts/update-yaml.py

```diff
@@ -37,12 +37,14 @@ def get_config(path):
             data = None
         else:
             raise e
+    else:
+        data = yaml.safe_load(data)
 
     if data is None:
         return {}
     else:
         # assume config is a dict
```

Comment:
Not sure I agree...we do in fact assume we are returning a dict, else we get the error that this issue was raised for.
Having said that I don't care that much one way or another, so I'm happy to just remove it.